### PR TITLE
Arodriguez/feature 157 report hazards select location fly to current location

### PR DESCRIPTION
**Pull request by ALFAROSO**
Hi, guys. This branch is to adding animations to the map. I encountered a conflict when using the 'hidden' class that prevented the map from fully loading. To address this, I suggest implementing an animation using CSS, HTML, and JavaScript while the DOM is loading.

### CODE REVIEW

File: pages/report-hazard/style.css [hunk 1]
```diff
@@ -1,3 +1,55 @@
+/* loadign animation */
```
Comment (whamester) [inline pages/report-hazard/style.css:1]: loadign typo

File: pages/report-hazard/style.css [hunk 2]
```diff
@@ -1,3 +1,55 @@
...
+.lds-hourglass {
+  display: inline-block;
```
Comment (whamester) [inline pages/report-hazard/style.css:3]: Let's you move this loader to a different component.

File: pages/report-hazard/style.css [hunk 3]
```diff
@@ -1,3 +1,55 @@
...
+  justify-content: center;
+  align-items: center;
+  position: fixed;
+  top: 0;
+  left: 0;
+  z-index: 10000;
+  clip-path: circle(150% at 100% 0);
+  transition: clip-path 1.9s ease-in-out;
+}
+.loader2 {
```
Comment (whamester) [inline pages/report-hazard/style.css:48]: let's remove the circle animation, just the white overlay is good for now

File: pages/report-hazard/style.css (hunk 1 above)
Comment (ALFAROSO) [reply pages/report-hazard/style.css:1]: Done

File: pages/report-hazard/style.css (hunk 2 above)
Comment (ALFAROSO) [reply pages/report-hazard/style.css:3]: Done

File: pages/report-hazard/style.css (hunk 3 above)
Comment (ALFAROSO) [reply pages/report-hazard/style.css:48]: Removed

**Pull request merged**